run.py

```diff
@@ -0,0 +1,3 @@
+#!flask/bin/python
+from app import app
+app.run(debug=False, host='10.11.12.170')
```

Comment:
Don't hardcode the hostname/IP address. When it gets deployed to production, it will have different values.